content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/index.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: 'MongoDB'
+metaTitle: 'Start from scratch with MongoDB (15 min)'
+metaDescription: 'Learn how to create a new Node.js or TypeScript project from scratch by connecting Prisma to your MongoDB database and generating Prisma Client for database access.'
+duration: '15 min'
+toc: false
+langSwitcher: ['typescript', 'node']
+dbSwitcher: ['mongodb']
+---
+
+<TopBlock>
+
+Learn how to create a new Node.js or TypeScript project from scratch by connecting Prisma to your MongoDB database and generating Prisma Client for database access. The following tutorial introduces you to the [Prisma CLI](../../../../concepts/components/prisma-cli), [Prisma Client](../../../../concepts/components/prisma-client), and [Prisma Migrate](../../../../concepts/components/prisma-migrate).
```

Comment:
```suggestion
Learn how to create a new Node.js or TypeScript project from scratch by connecting Prisma to your MongoDB database and generating Prisma Client for database access. The following tutorial introduces you to the [Prisma CLI](../../../../concepts/components/prisma-cli) and [Prisma Client](../../../../concepts/components/prisma-client).
```

Is there even a "tutorial" there though?

Reply:
Im unsure what you mean here? The tutorial is the same as the others we have. Are the buttons at the bottom not rendering for you? It should have a button that says `CONNECT YOUR DATABASE`

Reply:
This was also based on me not seeing the buttons. I am 100% convinced they did not render for me when I clicked through the preview :(

(The suggestion here is unrelated though and based on the fact that we do not actually support Migrate with MongoDB)